Fails to install on nodejs 4.2 and 5.10

Errors underneath wrtc dependency. I was however able to install `wrtc`, version 0.0.59 - so looks like something they have fixed already.

Full error follows

```
[jon@jon-laptop fbp-protocol-client]$ npm install --save rtc-plugin-node
npm WARN package.json fbp-protocol-client@0.1.6 No license field.
npm WARN deprecated graceful-fs@3.0.8: graceful-fs version 3 and before will fail on newer node releases. Please update to graceful-fs@^4.0.0 as soon as possible.
-
> wrtc@0.0.58 install /home/jon/contrib/code/fbp-protocol-client/node_modules/rtc-plugin-node/node_modules/wrtc
> node-pre-gyp install --fallback-to-build

gyp: /home/jon/.node-gyp/4.2.6/common.gypi not found (cwd: /home/jon/contrib/code/fbp-protocol-client/node_modules/rtc-plugin-node/node_modules/wrtc) while reading includes of binding.gyp while trying to load binding.gyp
gyp ERR! configure error 
gyp ERR! stack Error: `gyp` failed with exit code: 1
gyp ERR! stack     at ChildProcess.onCpExit (/home/jon/contrib/code/fbp-protocol-client/node_modules/rtc-plugin-node/node_modules/wrtc/node_modules/node-gyp/lib/configure.js:343:16)
gyp ERR! stack     at emitTwo (events.js:87:13)
gyp ERR! stack     at ChildProcess.emit (events.js:172:7)
gyp ERR! stack     at Process.ChildProcess._handle.onexit (internal/child_process.js:200:12)
gyp ERR! System Linux 4.4.5-1-ARCH
gyp ERR! command "/home/jon/.nvm/versions/node/v4.2.6/bin/node" "/home/jon/contrib/code/fbp-protocol-client/node_modules/rtc-plugin-node/node_modules/wrtc/node_modules/node-gyp/bin/node-gyp.js" "configure" "--fallback-to-build" "--module=/home/jon/contrib/code/fbp-protocol-client/node_modules/rtc-plugin-node/node_modules/wrtc/build/wrtc/v0.0.58/Release/node-v46-linux-x64/wrtc.node" "--module_name=wrtc" "--module_path=/home/jon/contrib/code/fbp-protocol-client/node_modules/rtc-plugin-node/node_modules/wrtc/build/wrtc/v0.0.58/Release/node-v46-linux-x64"
gyp ERR! cwd /home/jon/contrib/code/fbp-protocol-client/node_modules/rtc-plugin-node/node_modules/wrtc
gyp ERR! node -v v4.2.6
gyp ERR! node-gyp -v v1.0.3
gyp ERR! not ok 
node-pre-gyp ERR! build error 
node-pre-gyp ERR! stack Error: Failed to execute '/home/jon/.nvm/versions/node/v4.2.6/bin/node /home/jon/contrib/code/fbp-protocol-client/node_modules/rtc-plugin-node/node_modules/wrtc/node_modules/node-gyp/bin/node-gyp.js configure --fallback-to-build --module=/home/jon/contrib/code/fbp-protocol-client/node_modules/rtc-plugin-node/node_modules/wrtc/build/wrtc/v0.0.58/Release/node-v46-linux-x64/wrtc.node --module_name=wrtc --module_path=/home/jon/contrib/code/fbp-protocol-client/node_modules/rtc-plugin-node/node_modules/wrtc/build/wrtc/v0.0.58/Release/node-v46-linux-x64' (1)
node-pre-gyp ERR! stack     at ChildProcess.<anonymous> (/home/jon/contrib/code/fbp-protocol-client/node_modules/rtc-plugin-node/node_modules/wrtc/node_modules/node-pre-gyp/lib/util/compile.js:73:29)
node-pre-gyp ERR! stack     at emitTwo (events.js:87:13)
node-pre-gyp ERR! stack     at ChildProcess.emit (events.js:172:7)
node-pre-gyp ERR! stack     at maybeClose (internal/child_process.js:821:16)
node-pre-gyp ERR! stack     at Process.ChildProcess._handle.onexit (internal/child_process.js:211:5)
node-pre-gyp ERR! System Linux 4.4.5-1-ARCH
node-pre-gyp ERR! command "/home/jon/.nvm/versions/node/v4.2.6/bin/node" "/home/jon/contrib/code/fbp-protocol-client/node_modules/rtc-plugin-node/node_modules/wrtc/node_modules/.bin/node-pre-gyp" "install" "--fallback-to-build"
node-pre-gyp ERR! cwd /home/jon/contrib/code/fbp-protocol-client/node_modules/rtc-plugin-node/node_modules/wrtc
node-pre-gyp ERR! node -v v4.2.6
node-pre-gyp ERR! node-pre-gyp -v v0.6.4
node-pre-gyp ERR! not ok 
Failed to execute '/home/jon/.nvm/versions/node/v4.2.6/bin/node /home/jon/contrib/code/fbp-protocol-client/node_modules/rtc-plugin-node/node_modules/wrtc/node_modules/node-gyp/bin/node-gyp.js configure --fallback-to-build --module=/home/jon/contrib/code/fbp-protocol-client/node_modules/rtc-plugin-node/node_modules/wrtc/build/wrtc/v0.0.58/Release/node-v46-linux-x64/wrtc.node --module_name=wrtc --module_path=/home/jon/contrib/code/fbp-protocol-client/node_modules/rtc-plugin-node/node_modules/wrtc/build/wrtc/v0.0.58/Release/node-v46-linux-x64' (1)
npm ERR! Linux 4.4.5-1-ARCH
npm ERR! argv "/home/jon/.nvm/versions/node/v4.2.6/bin/node" "/home/jon/.nvm/versions/node/v4.2.6/bin/npm" "install" "--save" "rtc-plugin-node"
npm ERR! node v4.2.6
npm ERR! npm  v2.14.12
npm ERR! code ELIFECYCLE

npm ERR! wrtc@0.0.58 install: `node-pre-gyp install --fallback-to-build`
npm ERR! Exit status 1
npm ERR! 
npm ERR! Failed at the wrtc@0.0.58 install script 'node-pre-gyp install --fallback-to-build'.
npm ERR! This is most likely a problem with the wrtc package,
npm ERR! not with npm itself.
npm ERR! Tell the author that this fails on your system:
npm ERR!     node-pre-gyp install --fallback-to-build
npm ERR! You can get their info via:
npm ERR!     npm owner ls wrtc
npm ERR! There is likely additional logging output above.

npm ERR! Please include the following file with any support request:
npm ERR!     /home/jon/contrib/code/fbp-protocol-client/npm-debug.log
```

--- a/package.json
+++ b/package.json
@@ -24,7 +24,7 @@
   },
   "homepage": "https://github.com/rtc-io/rtc-plugin-node",
   "dependencies": {
-    "wrtc": "0.0.58"
+    "wrtc": "0.0.59"
   },
   "devDependencies": {
     "rtc-quickconnect": "^5.4.0",

